controllers/RatingsController.js

```diff
@@ -0,0 +1,76 @@
+import model from '../models';
+import StatusResponse from '../helpers/StatusResponse';
+
+const { ratings } = model;
+
+/**
+ * @description - This class is all about users ratings of articles
+ * @param {object} req
+ * @param {object} res
+ * @returns {class} Users
+ */
+class RatingsController {
+  /**
+   * @description - This method takes care of creating ratings on an article by a user
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} Users Ratings on an article
+   */
+  static async create(req, res) {
+    try {
+      const usersRatings = await ratings.create({
+        userId: req.userId,
+        identifier: req.params.identifier,
```

Comment:
Use the article Id here instead of identifier

Reply:
Noted @Chidi3b3r3